landing-page/content/about/about.html

```diff
@@ -23,8 +23,8 @@
 <div class="button-box">
 <ul class="list-inline intro-social-buttons">
     <li>
-        <a href="/getting-started" class="btn btn-default btn-lg">
-            <span class="network-name">Learn More</span>
+        <a href="/quickstarts" class="btn btn-default btn-lg">
+            <span class="network-name">Getting Started</span>
```

Comment:
This changes the button under "What is Iceberg?" to take you to the Quickstarts page instead of diving right into the docs site.
<img width="942" alt="Screen Shot 2022-05-17 at 3 41 52 PM" src="https://user-images.githubusercontent.com/43911210/168924095-c9a0a826-db04-4924-9017-b6a9745393c0.png">